invenio_communities/members/services/service.py

```diff
@@ -328,16 +328,27 @@ def _invite_factory(self, identity, community, role, visible, member, message, u
             )
 
     def search(
-        self, identity, community_id, params=None, search_preference=None, **kwargs
+        self,
+        identity,
+        community_id,
+        params=None,
+        search_preference=None,
+        extra_filter=None,
+        **kwargs
     ):
         """Search."""
+        # Apply extra filters
+        filter_ = dsl.Q("term", **{"active": True})
```

Comment:
I am not sure if it was meant here, see that there is a service method below which has similar filtering, I think this one was mean to search for all members, but maybe someone else can confirm as well

Reply:
To be more clear, it looks like that this method and the new `scan` below are exactly the same?

Reply:
Thank you. The `scan=True` parameter is missing in the `scan`method. Will add in a bit.

It discussed with @slint and @kpsherva to be more explicit about the usage of scan. So not only passing `scan=True` to the `search()` method but explicitly calling `scan()`

Reply:
lets discuss this during standup, I am not sure I understand why we need the scan=True parameter in the scan method.